tests/datasource/conftest.py

```diff
@@ -64,6 +69,9 @@ def test_cases_for_aws_glue_data_catalog_data_connector_spark_execution_engine(
     )
 
 
+@pytest.mark.skipif(
+    PYTHON_VERSION > Version("3.11"), reason="Fails on Python 3.12"
+)  # FIXME: before removing experimental status for Python 3.12
```

Comment:
@billdirks This is the only difference between this `0.18.x` version and the `1.0` version.
The glue moto mock seems to have issues on python 3.12